docs/site/content/docs/latest/sample.md

```diff
@@ -0,0 +1,87 @@
+
+# Deploy a Test Workload to a Workload Cluster
```

Comment:
This would also work for `unmanaged-cluster`s. Could we make this generic to include for both?